syntax = "proto3";

message RequestParams {
	bool user_list = 1;
}

message User {
	string first_name = 1;
	string last_name = 2;
	string email = 3;
	int32 age = 4;
}

message UserList {
	repeated User users = 1;
}

service UserService {
	rpc GetUserList(RequestParams) returns (UserList) {}
}
